docs/openapi.json

```diff
@@ -619,8 +619,8 @@
     },
     "/api/workflows/{workflow_id_or_name}/workspace": {
       "get": {
-        "description": "This resource is expecting a workflow UUID and a filename to return its list of code|input|output files.",
-        "operationId": "get_workflow_files",
+        "description": "This resource is expecting a workflow run UUID which workspace file list will be retrieved.",
```

Comment:
how about: "This resource retrieves the file list of a workspace, given its workflow run UUID."?